src/theme/ThemedComponents/Badge/LabelBadge.js

```diff
@@ -112,7 +112,7 @@ export const LabelBadge = ({label, variant = "default", style, ...rest}) => {
 
 LabelBadge.propTypes = {
     label: PropTypes.string.isRequired,
-    type: PropTypes.oneOf([
+    variant: PropTypes.oneOf([
```

Comment:
I like how the variant's proptypes set up here. I need to restrict the button's to do the same.